Doc/library/argparse.rst

```diff
@@ -2031,7 +2031,25 @@ Mutual exclusion
 
    Note that currently mutually exclusive argument groups do not support the
    *title* and *description* arguments of
-   :meth:`~ArgumentParser.add_argument_group`.
+   :meth:`~ArgumentParser.add_argument_group`. However, you can add a
+   mutually exclusive group to an argument group so it appears in the help
+   message with a title and description. For example::
+
+     >>> parser = argparse.ArgumentParser(prog='PROG')
+     >>> group = parser.add_argument_group('Group title', 'Group desciption')
+     >>> exclusive_group = group.add_mutually_exclusive_group(required=True)
+     >>> exclusive_group.add_argument('--foo', help='foo help')
+     >>> exclusive_group.add_argument('--bar', help='bar help')
+     >>> parser.print_help()
+     usage: PROG [-h] (--foo | --bar)
```

Comment:
Could you confirm this output? I'm getting this instead:

```
usage: PROG [-h] (--foo FOO | --bar BAR)

options:
  -h, --help  show this help message and exit
```

Reply:
Yeah, same. I must have copied this from above. Fixed, thank you.